tests/testutils/addon_helpers.py

```diff
@@ -14,6 +14,7 @@
 ADDON_WITH_BUNDLES_TYPE = "with_bundles"
 ADDON_WITH_IMAGESET_TYPE = "with_imageset"
 ADDON_WITH_INDEXIMAGE_TYPE = "with_indeximage"
+ADDON_WITH_SECRETS_TYPE = "with_secrets"
```

Comment:
```suggestion
ADDON_WITH_SECRETS_TYPE = "with_secrets"
```
we can get rid of this as this is semantically meant to represent the format which addon follows i.e. bundles (package manifest), index image or imageset.

For your test, feel free to use `ADDON_WITH_BUNDLES_TYPE` :)